documentation/modules/proc-configuring-kafka-bridge-tracing.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-bridge-config.adoc
+
+[id='proc-configuring-kafka-bridge-tracing-{context}']
+= Configuring distributed tracing
+
+[role="_abstract"]
+Enable distributed tracing to trace messages consumed and produced by the Kafka Bridge, and HTTP requests from client applications.
+
+Properties to enable tracing are present in the `application.properties` file.
+To enable distributed tracing, do the following:
+
+* Uncomment the required tracing properties in the properties file.
+* Set environment variables for tracing.
+
+With the default configuration, you can introduce a Jaeger tracing system based on OpenTelemetry or OpenTracing.
```

Comment:
wdym by "you can introduce" ?

Reply:
Changed to: _implement a Jaeger tracing system_

Reply:
As not a native speaker, it doesn't sound great to me as well. Of course, it could be just me.
I guess the concept you are trying to explain is something like: "with the default configuration, by using OpenTelemetry or OpenTracing, you will get tracing based on Jaeger implementation underneath". Does "you can introduce a Jaeger ..." or "implement a Jaeger ..." map to the above sentence? If yes ... then it's me not getting it :-) ... even if I would elaborate more.